src/models/expense.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const { sequelize } = require('../db.js');
+const { DataTypes } = require('sequelize');
+
+const Expenses = sequelize.define('Expenses', {
+  id: {
+    type: DataTypes.UUID,
+    defaultValue: DataTypes.UUIDV4,
+    primaryKey: true,
+  },
+  userId: {
+    type: DataTypes.UUIDV4,
+    primaryKey: true,
+  },
+  spentAt: {
+    type: DataTypes.DATE,
+    allowNull: false,
+  },
+  title: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  amount: {
+    type: DataTypes.NUMBER,
+    allowNull: false,
+  },
+  category: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  note: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  createdAt: {
+    type: DataTypes.DATE,
+    field: 'created_at',
```

Comment:
Use `underscored: false` option to avoid field renaming 

https://sequelize.org/docs/v6/other-topics/naming-strategies/#the-underscored-option